buildtest/buildsystem/checks.py

```diff
@@ -506,6 +506,61 @@ def assert_eq_check(builder):
     return all(assert_check)
 
 
+def assert_ne_check(builder):
+    """This method performs Assert not Equal and is used when ``assert_ne`` property is specified
+    in status check. This method will evaluate each metric value reference value and
+    store assertion in list. The list of assertion is logically AND which will return a True or False
+    for the status check.
+
```

Comment:
Please add a `args` section to describe the `builder` argument. you can copy from another method.

Reply:
@shahzebsiddiqui Done